requirements/dev.txt

```diff
@@ -2,3 +2,4 @@
 -r base.txt
 
 money-to-prisoners-common[testing]>=9.16,<9.17
+watchdog>=0.9.0,<1
```

Comment:
i don't get why this isn't installed automatically by mtp-common

Reply:
🤷‍♀ me neither, but working on the assumption that this will be skipped if previous installations are successful, then doesn't hurt.